### PR TITLE
Add Contact Us page

### PR DESCRIPTION
**Pull request by Idan-Garay**
Required:
- env variable "BREVO_CONTACT_TEMPLATE_ID" is required for this to work

Steps:
1. Visit /contact page
2. Fill up contact form
3. Click submit

Checks:
1. On submit, a new inquiry is saved in the mongodb collection inquiries under aintboard_dev
2. On submit, you should receive an email with title "We received your inquiry!"

### CODE REVIEW

File: app/contact/page.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+import Input from "components/Common/Input";
+import Label from "components/Common/Label";
+import Button from "components/Common/Button";
+import { InputContainer, ButtonContainer } from "components/Common/inputStyled";
+import { sendContactForm } from "util/email/sendContactForm";
+import { saveInquiry } from "db/saveInquiry";
+import { toast } from "react-toastify";
+
+// import { toast } from "react-toastify";
```
Comment (idiglove) [inline app/contact/page.tsx]: pls remove

File: app/contact/page.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        <div className="bg-darkGreenOpaque py-12">
+          <div className="py-8 px-12 max-w-xl mx-auto bg-darkestGreen rounded-lg min-h-fit">
+            <Formik
+              enableReinitialize
+              initialValues={{
+                email: "",
+                name: "",
+                message: "",
+              }}
+              // validationSchema={formSchema}
```
Comment (idiglove) [inline app/contact/page.tsx]: we dont validate?

File: app/contact/page.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                    name: values.name,
+                    message: values.message,
+                  });
+                  if (isSent) {
+                    toast.success(`Inquiry sent.`);
+                    resetForm();
+                  } else {
+                    toast.error(`Sending failed.`);
+                  }
+                  await saveInquiry({
```
Comment (idiglove) [inline app/contact/page.tsx:60]: good decision i think. didnt think of it this way for saving inquiry all the time

**Review by idiglove — APPROVED** [review]
just some comments then good to go

File: app/contact/page.tsx (hunk 2 above)
Comment (Idan-Garay) [reply app/contact/page.tsx]: I think we can validate the message's length? The input type email can detect if it's an email or not.

**Pull request merged**